### PR TITLE
GAWB-664: Add warning when about to launch a ton of WFs

### PR DESCRIPTION
**Pull request by MattPutnam**
This uses a new config, but since there's a default this code works even without the change to configs.

Also moved all data model specific code to one spot in common, in preparation for making the UI data model agnostic.
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] **Submitter**: Tell tech lead that the PR exists if s/he wants to look at it
- [x] **Submitter**: Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [x] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [x] **Tech lead**: sign off
- [x] **LR**: sign off
- [x] **Assign to submitter** to finalize
- [ ] **Submitter**: Squash commits, rebase if necessary
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Merge to develop 
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: Check configuration files in Jenkins in case they need changes
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't 
  working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev server still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by MattPutnam** [comment]
Is that config key name ok?

**Comment by davidangb** [comment]
At risk of scope creep, take a look at GAWB-701 and see how that ticket interacts with this PR? You're already adding onClick handlers for the entity types in the launch dialog, and 701 will also need to read from a "sample_set" -> "samples", "participant_set" -> "participants", "pair_set" -> "pairs" map/case/whatever.

File: src/cljs/org/broadinstitute/firecloud_ui/common.cljs [hunk 1]
```diff
@@ -131,3 +131,23 @@
...
+    1))
+
+(defn- make-pair [single-type]
+  [{:header (str "# " single-type) :starting-width 110}
+   #(count (get-in % ["attributes" single-type]))])
+
+(defn make-count-column
+  "When the given entity type is a *_set, returns [column_header fn-from-row-to-count]"
+  [entity-type]
+  (case entity-type
```
Comment (davidangb) [inline src/cljs/org/broadinstitute/firecloud_ui/common.cljs]: I think you should break the "sample_set" -> "samples", "participant_set" -> "participants", "pair_set" -> "pairs" mappings out into a separate value/function somewhere, and query that map from both count-workflows and make-count-column (as well as from GAWB-701 when that gets implemented).

**Comment by MattPutnam** [comment]
Yeah, I think 701 will do a little more restructuring, but it'll be easy to do from here.

**Comment by davidangb** [comment]
Looks pretty good to me, one comment inline about consolidating a map.

Also, if you're going to make a config change - even with a default value - please make PRs in firecloud-develop for local and dev.

(I'm fine with the config name, don't really have an opinion.)

**Comment by MattPutnam** [comment]
Yep, that firecloud-develop PR is incoming.  Will do the map thing.

**Comment by MattPutnam** [comment]
Related:
https://github.com/broadinstitute/firecloud-develop/pull/73

**Comment by davidangb** [comment]
Looks good! :+1:

**Pull request merged**